### PR TITLE
Cisco: Add modules, update M5/6

### PR DESCRIPTION
**Pull request by btriller**
* Add Cisco UCS VICs 1400/14000/15000 [1] [2]
* Update PCIe/mRAID/mLOM positions

[1] https://www.cisco.com/c/en/us/products/collateral/interfaces-modules/unified-computing-system-adapters/datasheet-c78-741130.html
[2] https://www.cisco.com/c/en/us/products/collateral/interfaces-modules/unified-computing-system-adapters/ucs-vic-15000-series-ds.html

### CODE REVIEW

File: module-types/Cisco/UCSC-9400-8E.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+manufacturer: Cisco
+model: 12G 9400-8e HBA
+part_number: UCSC-9400-8E
+comments: Cisco 12G 9400-8e HBA for external JBOD attach
```
Comment (harryajc) [inline module-types/Cisco/UCSC-9400-8E.yaml:5]: Right now, storage and raid controllers should not be modelled in this repo

File: module-types/Cisco/UCSC-9500-8E.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+manufacturer: Cisco
+model: 9500-8e 12G SAS HBA
+part_number: UCSC-9500-8E
+comments: Cisco 9500-8e 12G SAS HBA for external JBOD attach
```
Comment (harryajc) [inline module-types/Cisco/UCSC-9500-8E.yaml:5]: Right now, storage and raid controllers should not be modelled in this repo

File: module-types/Cisco/UCSC-RAID-M5.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+manufacturer: Cisco
+model: 12G Modular RAID Controller with 2-GB cache
+part_number: UCSC-RAID-M5
+comments: Cisco 12G Modular RAID Controller with 2-GB cache
```
Comment (harryajc) [inline module-types/Cisco/UCSC-RAID-M5.yaml:5]: Right now, storage and raid controllers should not be modelled in this repo

File: module-types/Cisco/UCSC-RAID-M6T.yaml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+manufacturer: Cisco
+model: 12G SAS RAID Controller with 4GB FBWC
+part_number: UCSC-RAID-M6T
+comments: Cisco 12G SAS RAID Controller with 4GB FBWC (16 Drives)
```
Comment (harryajc) [inline module-types/Cisco/UCSC-RAID-M6T.yaml:5]: Right now, storage and raid controllers should not be modelled in this repo

File: module-types/Cisco/UCSC-SAS-M5.yaml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
+manufacturer: Cisco
+model: 12G Modular SAS HBA
+part_number: UCSC-SAS-M5
+comments: Cisco 12G Modular SAS HBA
```
Comment (harryajc) [inline module-types/Cisco/UCSC-SAS-M5.yaml:5]: Right now, storage and raid controllers should not be modelled in this repo

File: module-types/Cisco/UCSC-SAS-M6T.yaml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+---
+manufacturer: Cisco
+model: 12G SAS HBA
+part_number: UCSC-SAS-M6T
+comments: Cisco 12G SAS HBA (16 Drives)
```
Comment (harryajc) [inline module-types/Cisco/UCSC-SAS-M6T.yaml:5]: Right now, storage and raid controllers should not be modelled in this repo

**Review by harryajc — CHANGES_REQUESTED** [review]

**Comment by btriller** [comment]
Removed storage/RAID controllers

**Review by harryajc — APPROVED** [review]

**Pull request merged**